app/lib/common/widgets/error_widget.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+
+class ErrorWidgetTemplate extends StatelessWidget {
```

Comment:
```suggestion
class EmptyStateWidgetTemplate extends StatelessWidget {
```
- Better to not use `Error word` as `ErrorState` and `EmptyState` are different things.